src/main/java/gmail/taras/shevchuk0024/lesson40_Functional_Interface/Function.java

```diff
@@ -0,0 +1,5 @@
+package gmail.taras.shevchuk0024.lesson40_Functional_Interface;
+
+public class Function {
+    public static final java.util.function.Function<Double, Long> roundToLong = d -> Math.round(d);
+}
```

Comment:
Статичний метод вже по факту фінальний тому можна це опустити)

```suggestion
package gmail.taras.shevchuk0024.lesson40_Functional_Interface;

import java.util.function.Function;

public class FunctionUtils {
    public static Function<Double, Long> roundToLong = d -> Math.round(d);
}
```